.github/workflows/ci.yml

```diff
@@ -0,0 +1,69 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request: ~
+
+jobs:
+  # derived from https://dev.to/felipetofoli/github-actions-for-net-full-framework-build-and-test-299h
+  NET_Tests:
+    runs-on: windows-2019
+    strategy:
+      matrix:
+        dotnet-version: [ '35', '40' ]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Setup variables
+        id: test_file
+        run: echo "::set-output name=test_file::EasyPost.Tests.Net${{ matrix.dotnet-version }}"
+      - name: Setup MSBuild
+        if: always()
+        uses: microsoft/setup-msbuild@v1.1
+      - name: Setup Nuget
+        uses: NuGet/setup-nuget@v1.0.5
```

Comment:
# Reviewer Note

This action does not setup the other various Visual Studio tools which is why we need all the separate actions here.